src/Compilers/VisualBasic/Portable/Binding/Binder_Initializers.vb

```diff
@@ -178,13 +178,13 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                 End If
                             End If
 
-                            initializerBinder.BindFieldInitializer(initializer.FieldsOrProperties,
+                            initializerBinder.BindFieldInitializer(initializer.FieldsOrProperties.Cast(Of FieldSymbol).ToImmutableArray(),
```

Comment:
Is this using IEnumerable.Cast? Consider using ToDowncastedImmutable helper instead.